metashare/multisalesforce/views.py

```diff
@@ -87,7 +87,7 @@ def parse_token(self, data):
 
         Called from OAuth2CallbackView"""
         data["access_token"] = fernet_encrypt(data["access_token"])
-        data["refresh_token"] = fernet_encrypt(data["refresh_token"])
+        # data["refresh_token"] = fernet_encrypt(data["refresh_token"])
```

Comment:
Why are we not storing the refresh token?

Reply:
Because I was debugging something and left that in there by mistake.